app/src/main/java/tech/relaycorp/gateway/App.kt

```diff
@@ -138,6 +139,12 @@ open class App : Application() {
         }
     }
 
+    protected open fun deleteExpiredCertificates() {
+        backgroundScope.launch {
+            localConfig.bootstrap()
```

Comment:
Wrong method. I think it'd be good to add a test for this.

Reply:
🤦

Reply:
Unfortunately, this is not a very easy test to write. App instances are weird beasts. The easiest way would be to move all these initializations to another class that gets called by the App. But that probably is a better fit for another task.